src/oc/plot.cpp

```diff
@@ -866,9 +866,7 @@ void vtplot(int mode, double x, double y) {
 #endif /*VT*/
 
 int set_color(int c) {
-    if (c >= 0 || c < 128) {
-        hoc_color = c;
-    }
```

Comment:
funny one, the people mean && instead of ||, right?

Reply:
It's been like this for a long time: https://github.com/neuronsimulator/nrn/commit/1f6563798a17780d2dc3b42812e2a58189809a3c#diff-809866e9965191b593534ecdda3c445550482750f366ff725c3f5baca21141f9R1022 🙈